.github/workflows/push.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: |
           echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
           echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
-          go install gotest.tools/gotestsum@latest
+          go install gotest.tools/gotestsum@v1.12.0
```

Comment:
Is this something that dependabot can upgrade?

Otherwise maybe it's better to add this to build dependencies as explained here? https://rednafi.com/go/omit_dev_dependencies_in_binaries/

Reply:
Dependabot won't work here.

If we track tool deps somewhere, I'd rather do it in a separate `go.mod` to keep the main one clean.

Reply:
Added this to the list of things to discuss tomorrow.